pipeline/filters/kubernetes.md

```diff
@@ -427,60 +423,58 @@ And if you are in debug mode, you could see more:
 
 ## Troubleshooting
 
-The following section goes over specific log messages you may run into and how to solve them to ensure that Fluent Bit's Kubernetes filter is operating properly
+The following section goes over specific log messages you might receive. Learn how to solve them to ensure that the Fluent Bit Kubernetes filter is operating properly
 
-### I can't see metadata appended to my pod or other Kubernetes objects
+- You can't see metadata appended to your pods or other Kubernetes objects
 
-If you are not seeing metadata added to your kubernetes logs and see the following in your log message, then you may be facing connectivity issues with the Kubernetes API server.
+  If you aren't seeing metadata added to your Kubernetes logs and see the following in your log message, then you might be facing connectivity issues with the Kubernetes API server.
 
-```text
-[2020/10/15 03:48:57] [ info] [filter_kube] testing connectivity with API server...
-[2020/10/15 03:48:57] [error] [filter_kube] upstream connection error
-[2020/10/15 03:48:57] [ warn] [filter_kube] could not get meta for POD
-```
+  ```text
+  [2020/10/15 03:48:57] [ info] [filter_kube] testing connectivity with API server...
+  [2020/10/15 03:48:57] [error] [filter_kube] upstream connection error
+  [2020/10/15 03:48:57] [ warn] [filter_kube] could not get meta for POD
+  ```
 
-**Potential fix \#1: Check Kubernetes roles**
+  - Potential fix #1: Check Kubernetes roles
```

Comment:
NB: Avoid `#`, which is unnecessary here and can be problematic in Markdown

```suggestion
  - Potential fix 1: Check Kubernetes roles
```

Reply:
ugh I thought i removed those